annotations/update_pending_annotations.py

```diff
@@ -0,0 +1,152 @@
+import synapseclient
+from synapseclient import Table, RowSet
+import argparse
+import pandas as pd
+from attribute_dictionary import PUBLICATION_DICT
+
+
+### Login to Synapse ###
+def login():
+
+    syn = synapseclient.Synapse()
+    syn.login()
+
+    return syn
+
+
+def get_args():
+
+    parser = argparse.ArgumentParser(
+        description=
+        'Get synapse table id of annotations to be editd and synapse table id of controlled vocabulary mappings'
```

Comment:
It's best to limit lines of code to a max number of characters, depending on the workplace standards.  The reason for this max character limit is to prevent horizontal scrolling when reading/writing code (especially if someone is using a smaller screen).

I personally use 79 characters as the cut-off, but I think Sage, in general, uses 88 characters.

So, for this one, you can split the string across multiple lines, e.g.

```
    parser = argparse.ArgumentParser(
        description='Get synapse table id of annotations to be editd and '
                    'synapse table id of controlled vocabulary mappings'
    )
```

Reply:
Is there something I can install on vis studio to do this for me? I think I did install it, but it only works for lines of code and not the strings. Do you count this manually?

Reply:
No, I use the bar at the bottom to see which character I'm on! e.g. this indicates I'm on line 16, character 79:

![Screen Shot 2022-11-21 at 10 23 30 AM](https://user-images.githubusercontent.com/9377970/203131471-ff54f9fc-45d0-49f1-ab23-8cac11e13bdd.png)